templatestore/views.py

```diff
@@ -66,6 +66,7 @@ def render_template_view(request):
 
     try:
         data = json.loads(request.body)
+        print(data)
```

Comment:
Remove print statements. Try adding logs if the data could be required for future references.